<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>691</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>add customer</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>511</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D9D9D9;
color: rgb(6, 6, 6);</string>
      </property>
      <property name="title">
       <string>add</string>
      </property>
      <widget class="QLineEdit" name="name_edit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="last_name_edit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9
</string>
       </property>
       <property name="placeholderText">
        <string>last name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="adress_edit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phone_edit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>phone number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_edit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>e-mail</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
background-color:#7088A6;

</string>
       </property>
       <property name="text">
        <string>addd</string>
       </property>
      </widget>
      <zorder>name_edit</zorder>
      <zorder>last_name_edit</zorder>
      <zorder>adress_edit</zorder>
      <zorder>email_edit</zorder>
      <zorder>pushButton</zorder>
      <zorder>phone_edit</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="update">
     <attribute name="title">
      <string>update informations</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>511</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D9D9D9;
color: rgb(6, 6, 6);</string>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QLineEdit" name="name_edit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="last_name_edit_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>last name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="adress_edit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phone_edit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
       </property>
       <property name="placeholderText">
        <string>phone number</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email_edit_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-color: rgb(255, 255, 255);
border-radius:unset;
border-bottom:1px solid #d9d9d9
</string>
       </property>
       <property name="placeholderText">
        <string>e mail</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:8px;
background-color:#7088A6;
</string>
       </property>
       <property name="text">
        <string>modify</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="cust_list">
     <attribute name="title">
      <string>customers list </string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>21</y>
        <width>611</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="delete_2">
     <attribute name="title">
      <string>delete customer </string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius:unset;
border-bottom:1px solid #d9d9d9</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>insert phone number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PushButton1">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:8px;
background-color:#7088A6;
</string>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>531</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#D9D9D9;
color: rgb(6, 6, 6);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>lineEdit</zorder>
     <zorder>PushButton1</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
